src/System.Net.Http.WinHttpHandler/src/Resources/Strings.resx

```diff
@@ -121,4 +121,7 @@
   <data name="net_http_content_stream_already_read" xml:space="preserve">
     <value>The stream was already consumed. It cannot be read again.</value>
   </data>
+  <data name="PlatformNotSupported_WinHttpHandler" xml:space="preserve">
+    <value>WinHttpHandler is designed to be used only on .NET Framework or .NET Core platforms.  It is not designed to be used for Windows Store Applications (UWP)  nor Linux platforms.</value>
```

Comment:
Nits:
- There's an extra space before "nor"
- "Linux" => "Unix"... this applies to macOS as well
- Do we not consider UWP to be a .NET Core platform?
- "on .NET Framework or .NET Core platforms" => "on .NET Framework and .NET Core runtimes on Windows"
- "It is not designed to be used for" => "It is not supported on"